mpfdtype/mpfdtype/src/mpfdtype_main.c

```diff
@@ -47,10 +44,14 @@ PyInit__mpfdtype_main(void)
         goto error;
     }
 
-    if (init_mpf_umath() < 0) {
+    PyObject *numpy = init_mpf_umath();
+    
+    if (numpy == NULL) {
         goto error;
     }
 
+    Py_DECREF(numpy);
```

Comment:
Not sure I understand the `numpy` return change here.

Reply:
This was because I wasn't aware that there is a unique symbol macro for the ufunc API. Now I'm using that too and putting the import in the module init.